arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -714,7 +721,9 @@ Result RocksDBCollection::truncate(transaction::Methods& trx, OperationOptions&
 
   // avoid OOM error for truncate by committing earlier
   uint64_t const prvICC = state->options().intermediateCommitCount;
-  state->options().intermediateCommitCount = std::min<uint64_t>(prvICC, 10000);
+  auto const tmp = std::min<uint64_t>(prvICC, 10000);
+  state->options().intermediateCommitCount = tmp;
+  _statistics._numWrite.count(tmp);
```

Comment:
This does not make sense to me. We are setting the intermediate commit value here, but this is not a write operation that should be counted. It is just a (temporary) change of settings.

Reply:
now separately as truncations